contracts/modules/LoanClosings/LoanClosings.sol

```diff
@@ -90,7 +90,7 @@ contract LoanClosings is LoanClosingsBase {
         address receiver,
         uint256 swapAmount, // denominated in collateralToken
         bool returnTokenIsCollateral, // true: withdraws collateralToken, false: withdraws loanToken
-        bytes memory /*loanDataBytes*/) // for future use
+        bytes memory loanDataBytes) // for future use
```

Comment:
please describe structure of loanDataBytes in our IBZRx interface file in detail. current features and how to extend to more futures